services/nodemailer.js

```diff
@@ -0,0 +1,18 @@
+const MemoRoom = require("../models/MemoRoom");
+const User = require("../models/User");
+
+exports.verifyUser = async (email) => {
+  const user = await User.findOne({ email: email }).lean().exec();
+
+  return user;
```

Comment:
```suggestion
  return await User.findOne({ email: email }).lean().exec();
```
별도의 변수 선언 없이 바로 return 하시는것 어떠신가요??

Reply:
간결해져서 더 좋을것같아요! 수정해보겠습니다!